src/packages/core/property-editor/uis/dropdown/property-editor-ui-dropdown.element.ts

```diff
@@ -1,22 +1,65 @@
-import { html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
-import { UmbTextStyles } from "@umbraco-cms/backoffice/style";
+import { UUISelectEvent } from '@umbraco-ui/uui';
+import type { UmbPropertyEditorConfigCollection } from '@umbraco-cms/backoffice/property-editor';
+import { html, customElement, property, state } from '@umbraco-cms/backoffice/external/lit';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
 import { UmbPropertyEditorExtensionElement } from '@umbraco-cms/backoffice/extension-registry';
+import '../../../components/input-dropdown/input-dropdown-list.element.js';
```

Comment:
This import path looks off somehow. Do we have an alias for this perhaps?

Reply:
Eh, it's not even needed I just forgot to export the input element.